test/rules_test.js

```diff
@@ -180,27 +179,27 @@ describe('applyRules', () => {
     });
     it('multiply fractions', () => {
         const tests = [
-            ['2 / 3 * 2 / 3', '2 * 2 / 3 * 3'],
-            ['x / 2 * x / 2', 'x * x / 2 * 2'],
-            ['(x + 1) / 2 * (x + 1) / 2', '(x + 1) * (x + 1) / 2 * 2'],
-            ['x^((x + 1) / 2 * (x + 1) / 2)', 'x^(x + 1) * (x + 1) / 2 * 2'],
+            ['2 / 3 * 2 / 3', '(2 * 2) / (3 * 3)'],
+            ['x / 2 * x / 2', '(x * x) / (2 * 2)'],
+            ['(x + 1) / 2 * (x + 1) / 2', '((x + 1) * (x + 1)) / (2 * 2)'],
+            ['x^((x + 1) / 2 * (x + 1) / 2)', 'x^((x + 1) * (x + 1)) / (2 * 2)'],
         ];
         tests.forEach(t => assert.equal(applyRuleString(rules.MULTIPLY_FRACTIONS, t[0]), t[1]));
     });
-    it('simplfy division', () => {
+    it('simplify division', () => {
         const tests = [
-            ['2 / 3 / 4', '2 / 3 * 4'],
-            ['x / 2 / 2', 'x / 2 * 2'],
-            ['(x + 1) / 2 / (x + 1)', '((x + 1)) / 2 * (x + 1)'],
-            ['x^((x + 1) / 2 / 2)', 'x^((x + 1)) / 2 * 2'],
+            ['2 / 3 / 4', '2 / (3 * 4)'],
+            ['x / 2 / 2', 'x / (2 * 2)'],
+            ['(x + 1) / 2 / (x + 1)', '(x + 1) / (2 * (x + 1))'],
+            ['x^((x + 1) / 2 / 2)', 'x^(x + 1) / (2 * 2)'],
```

Comment:
Here is the AST for that test case: 
<img width="343" alt="screen shot 2017-05-01 at 11 44 27 pm" src="https://cloud.githubusercontent.com/assets/16139634/25603268/2e5560e6-2ec8-11e7-90cf-52d7ce8e3076.png">